tests/datasource/fluent/integration/test_sql_datasources.py

```diff
@@ -227,14 +225,14 @@ def __call__(
         ...
 
 
-@pytest.fixture(scope="function")
-def table_factory(
-    capture_engine_logs: pytest.LogCaptureFixture,
-    silence_sqla_warnings: None,  # TODO: remove this
-) -> Generator[TableFactory, None, None]:
+@pytest.fixture(
+    scope="class",
+)
+def table_factory() -> Generator[TableFactory, None, None]:
```

Comment:
I made this class scoped so we aren't re-creating the tables and columns every test.